dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -408,7 +408,7 @@ All providers (including overriding documentation for doc-only changes):
 ```shell script
 cd "${AIRFLOW_REPO_ROOT}"
 
-./docs/publish_docs.py \
+breeze release-management publish-docs \
```

Comment:
Do we need the whole section above?
Now that the command in breeze we probably don't need the above section?

```

**NOTE** In order to run the publish documentation you need to activate virtualenv where you installed
apache-airflow with doc extra:
* `pip install '.[doc_gen]'`
If you don't have virtual env set you can do:
```shell script
cd <path_you_want_to_save_your_virtual_env>
virtualenv providers
source venv/providers/bin/activate
pip install 'apache-airflow[doc_gen]'
```

```

Reply:
cool.
Please note that lines 432-434 also needs to be edited.
It's alternative approach to run the publishing command

Reply:
oh yeah. I missed that we had a bit more details in PROVIDERS docs :)

Reply:
I do not think we need to edit lines 432-434. Those changes have been handled in this PR already

Reply:
@eladkal @potiuk the PR having this fix is here: https://github.com/apache/airflow/pull/32559

Reply:
But why do we need bash script to being with?
Its executing the same command with differnt kind of parameters to breeze should do the mitigation

Reply:
It's a helper script converting list of ids which you already have (amazon google ... ) to list of flags (--package-filter apache-airlfow-providers-amazon --package-filter apache-airlfow-providers-google ....) .

If you prefer to do it manually fine, but I think having script is useful. We could of course add it to the command but --package-filter is traditionally there and serves other purposes too (not everything is a provider for one).